test/benchmarks.jl

```diff
@@ -70,20 +139,36 @@ Test on Moduli functions
 
 Test on the Maxwell model
 
-  67.689 ns (2 allocations: 112 bytes)
-  7.500 μs (3 allocations: 8.12 KiB)
-  35.061 ns (1 allocation: 16 bytes)
-  6.811 μs (1 allocation: 8.00 KiB)
+  67.570 ns (2 allocations: 112 bytes)
+  7.409 μs (3 allocations: 8.12 KiB)
+  33.446 ns (1 allocation: 16 bytes)
+  7.451 μs (1 allocation: 8.00 KiB)
 
 Test on the FractSLS_Zener model relaxation modulus - with mittleff
 
 Direct function evaluation
-  207.834 ns (7 allocations: 112 bytes)
-  67.608 μs (15 allocations: 8.53 KiB)
-From the RheoModel(Class)
-  169.915 ns (4 allocations: 160 bytes)
-  65.957 μs (3 allocations: 8.16 KiB)
-  103.534 ns (1 allocation: 16 bytes)
-  59.479 μs (1 allocation: 8.00 KiB)
-
+  209.077 ns (7 allocations: 112 bytes)
+  68.211 μs (15 allocations: 8.53 KiB)
+From the RheoModelClass
+  168.912 ns (4 allocations: 160 bytes)
+  64.546 μs (3 allocations: 8.16 KiB)
+From the RheoModel
+  105.382 ns (1 allocation: 16 bytes)
+  59.494 μs (1 allocation: 8.00 KiB)
+
+Test on the FractSLS_Zener model creep compliance - with inverse laplace
+
+
+function wrappers using Weeks
```

Comment:
Hi Louis, 
I wonderd as well what the appropriate location should be.
My logic here is that if comparing with RHEOS is part of the test, then we should link the benchmark to a specific commit of Rheos, and it may be best to keep the file in the repos, albeit not necessarily permanently.

My intention is to compare Talbot and Weeks with RHEOS, and monitor the performance as we develop it further. I am indeed considering implementing Weeks for arrays as part of RHEOS and then it would be relevant to keep the raw version as part of the benchmark. Having said that, I agree with you that many of these tests don't really fit here on the long term, at least  with this level of detail, and I will clear this up before the actual merge. I'd just keep a link to the relevant commit in the discussion/issue, so that more specific benchmark file is not lost for future reference. Does that make sense?

Reply:
I'm still not sure I fully understand the intention of `benchmarks.jl` but if it's helpful to your development process and it can't be in `RheoBenchandTest`, or a permanent separate branch of RHEOS, then I won't be a tough :cookie: about it :+1: